python/tests/io/test_pyarrow.py

```diff
@@ -411,3 +434,140 @@ def test_list_type_to_pyarrow():
         element_required=True,
     )
     assert visit(iceberg_map, _ConvertToArrowSchema()) == pa.list_(pa.int32())
+
+
+@pytest.fixture
+def bound_reference(table_schema_simple: Schema) -> BoundReference[str]:
+    return BoundReference(table_schema_simple.find_field(1), table_schema_simple.accessor_for_field(1))
+
+
+@pytest.fixture
+def bound_double_reference(table_schema_simple: Schema) -> BoundReference[float]:
+    schema = Schema(
+        NestedField(field_id=1, name="foo", field_type=DoubleType(), required=False),
+        schema_id=1,
+        identifier_field_ids=[2],
+    )
```

Comment:
Should we have a fixture that has a double field?

Reply:
Usually I split out things into fixtures if we use it at multiple places